client/my-sites/site-settings/settings-reading/main.tsx

```diff
@@ -20,11 +62,7 @@ const ReadingSettings = () => {
 		<Main className="site-settings">
 			<DocumentHead title={ translate( 'Reading Settings' ) } />
 			<FormattedHeader brandFont headerText={ translate( 'Reading Settings' ) } align="left" />
```

Comment:
In a similar way as above we could consider adding the class to the `FormattedHeader` component:

```suggestion
			<FormattedHeader brandFont className="settings-reading__page-heading" headerText={ translate( 'Reading Settings' ) } align="left" />
```